build-image/Dockerfile

```diff
@@ -1,9 +1,14 @@
 FROM golang:1.13.3-stretch
-RUN apt-get update && apt-get install -y python-requests python-yaml file jq unzip protobuf-compiler libprotobuf-dev && \
+RUN apt-get update && apt-get install -y curl python-requests python-yaml file jq unzip protobuf-compiler libprotobuf-dev && \
 	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN curl -sL https://deb.nodesource.com/setup_6.x | sh -
+RUN apt-get install -y nodejs npm && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN npm install -g postcss-cli autoprefixer
 RUN go clean -i net && \
 	go install -tags netgo std && \
 	go install -race -tags netgo std
+ENV HUGO_VERSION=v0.59.1
+RUN git clone https://github.com/gohugoio/hugo.git && cd hugo && git checkout ${HUGO_VERSION} && go install --tags extended
```

Comment:
Can this be done with a single `git install` command?
Alternatively `git clone --branch ${HUGO_VERSION} --depth 1` would be more efficient since you don't need the whole repo.

Reply:
trying !

Reply:
it works will push